manager/container_test.go

```diff
@@ -474,3 +474,51 @@ func TestParsePsLine(t *testing.T) {
 		})
 	}
 }
+
+var cgroupCases = []struct {
+	name    string
```

Comment:
can we add case here case where `cgroups==foobar` (some random string that doesn't match any of the regex / cases) and ensure that "/" is returned?

Reply:
thanks!